scripts/gas-refund-program/persistance/db-persistance.ts

```diff
@@ -160,29 +142,32 @@ export const saveMerkleTreeInDB = async ({
   chainId,
   epoch,
   merkleTree,
+  addressRefundedAmountPSP,
 }: {
   epoch: number;
   chainId: number;
   merkleTree: MerkleTreeData;
+  addressRefundedAmountPSP: Record<string, BigNumber>;
 }): Promise<void> => {
   const {
     root: { totalAmount, merkleRoot },
     leaves,
   } = merkleTree;
 
-  const epochDataToUpdate: CompletedEpochGasRefundData[] = leaves.map(
+  const epochDataToUpdate: GasRefundParticipantData[] = leaves.map(
     (leaf: MerkleData) => ({
       epoch,
       address: leaf.address,
       chainId: chainId,
 
       merkleProofs: leaf.merkleProofs,
       isCompleted: true,
+      refundedAmountPSP: addressRefundedAmountPSP[leaf.address].toFixed(0),
```

Comment:
I don't think we should store this value at all.
As it's always recomputable from GasRefundTransactions (again aggregation). 
The risk is if mess up the calculations, and end up with a mismatch between txs and refundParticipations, we might have hard time debugging. 
At the end of day, I'm always in favor of non duplication.